providers/oidc.go

```diff
@@ -98,7 +94,8 @@ func (p *OIDCProvider) Redeem(ctx context.Context, redirectURL, code, codeVerifi
 // EnrichSession is called after Redeem to allow providers to enrich session fields
 // such as User, Email, Groups with provider specific API calls.
 func (p *OIDCProvider) EnrichSession(ctx context.Context, s *sessions.SessionState) error {
-	err := p.enrichFromIntrospectURL(ctx, s)
+
+	err := p.PicsEnrichFromIntrospectURL(ctx, s)
```

Comment:
Just to be sure, you checked that this function is in the correct scope to call the PicsEnrichFromIntrospectURL from pics_oidc, right?

Reply:
Yes, both are in the same package providers